### PR TITLE
completed javascript-operators-and-expressions exercise

### PR DESCRIPTION
**Pull request by joshuaLeeT**
image

### CODE REVIEW

File: prep-javascript-operators-and-expressions/main.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/*       multiply                          */
+
+var number1 = 5;
```
Comment (BrettAlbright) [inline prep-javascript-operators-and-expressions/main.js]: make sure to replace all of the `var` declarations with `const` except for the last one as noted in the instructions, that should be declared with `let`

File: prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# prep-javascript-operators-and-expressions-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
```
Comment (BrettAlbright) [inline prep-javascript-operators-and-expressions/prep-javascript-operators-and-expressions-quiz-notes.md:6]: also need to answer the quiz questions

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Josh, make sure to update the variable declarations, and to answer the quiz questions

**Comment by joshuaLeeT** [comment]
completed quiz

**Pull request merged**